data/south jobs.txt

```diff
@@ -0,0 +1,213 @@
+# Copyright (c) 2017 by Michael Zahniser
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+mission "Pirate Occupation [0]"
+	name `Pirates occupying <system>`
+	minor
+	repeat
+	description `A small band of pirates have been occupying <system>, and the local government has asked for help. Defeat the pirates in the system and land on <planet> when done. Payment is <payment>.`
+	to offer
+		random < 15
+		"combat rating" > 100
+	source
+		attributes "south" "rim"
+	destination
+		distance 2 3
+		attributes "south"
+	npc
+		government "Militia"
+		personality heroic
+		fleet "Small Militia"
+	npc
+		government "Militia"
+		personality heroic staying
+		system destination
+		fleet "Small Militia"
+	npc evade
+		government "Pirate"
+		personality heroic staying target harvests plunders
+		system destination
+		fleet "Small Southern Pirates" 3
+	on offer
+		conversation
+			`Suddenly you hear raised voices and shouting outside as pilots run for their ships: "<destination> is under attack by a small pirate fleet! They are requesting help from any combat-worthy ship in the region to join in the defenses!" The authorities of <planet> will probably pay you quite well if you assist them, but this could also be an easy way to get yourself killed.`
```

Comment:
#3468 at minimum

Also, consider defining a stock `conversation` (see #3384). I note there are slight differences (small / ___ / large) but perhaps you are fine with dropping that.

Is it possible for the destination planets here to actually be very capable of defending themselves (e.g. a Free Worlds naval base?) `attributes south`, 2-3 jumps from `south` or `rim` doesn't seem very specific.

Reply:
Part of the whole Southern Defense Pact/FW deal is that worlds defend one another. The Rim is less susceptible to piracy due to how narrow the entrance is, but southern worlds are all rather close to the pirate planets of the south. A planet may be able to defend itself, but you wouldn't hear about it then, now would you. :)